addons/test_event_full/tests/test_website_event.py

```diff
@@ -0,0 +1,71 @@
+
+    @users('user_event_web_manager')
+    def test_seo_data(self):
+        """Test SEO data for submenus on event website page"""
+
+        # Create an event
+        event = self.env['event.event'].create({
+            'name': 'TestEvent',
+            'date_begin': fields.Datetime.to_string(datetime.today() + timedelta(days=1)),
+            'date_end': fields.Datetime.to_string(datetime.today() + timedelta(days=15)),
+            'website_menu': True,
+            'booth_menu': True,
+            'exhibitor_menu': True,
+            'community_menu': True,
+            'website_track': True,
+            'website_track_proposal': True,
+        })
+
+        self.assertFalse(event.website_meta_title, "Event should initially have no meta title")
+        event.write({
+            'website_meta_title': "info",
+        })
+        self.assertTrue(event.website_meta_title, "Event should have a meta title after writing")
+
+        def check_seo_data(page_name, meta_title):
+            view_model = self.env['ir.ui.view'].search([('name', '=', f'{page_name} TestEvent')])
+            self.assertFalse(view_model.website_meta_title, f"{page_name} page should initially have no meta title")
+
+            view_model.write({
+                'website_meta_title': meta_title,
+            })
+
+            self.assertTrue(view_model.website_meta_title, f"{page_name} page should have a meta title after writing")
+            self.assertNotEqual(event.website_meta_title, view_model.website_meta_title, f"Event and {page_name} page should have different meta titles")
+
+            return view_model
+
+        # Run SEO tests and store view models for different pages
+        view_model_booth = check_seo_data('Get A Booth', 'booth')
+        view_model_exhibitor = check_seo_data('Exhibitors', 'exhibitors')
+        view_model_community = check_seo_data('Community', 'community')
+        view_model_track = check_seo_data('Talks', 'talks')
+        view_model_agenda = check_seo_data('Agenda', 'agenda')
+        view_model_track_proposal = check_seo_data('Talk Proposals', 'talk proposals')
+
+        # SEO data test of submenus
+        view_models = [
+            (view_model_booth, "Booth page"),
+            (view_model_exhibitor, "Exhibitor page"),
+            (view_model_community, "Community page"),
+            (view_model_track, "Talks page"),
+            (view_model_agenda, "Agenda page"),
+            (view_model_track_proposal, "Talk Proposals page")
+        ]
+
+        # Compare each view model with every other view model to ensure their meta titles are different
+        for i in range(len(view_models)):
+            for j in range(i + 1, len(view_models)):
+                self.assertNotEqual(
+                    view_models[i][0].website_meta_title,
+                    view_models[j][0].website_meta_title,
+                    f"{view_models[i][1]} and {view_models[j][1]} should have different meta titles"
+                )
```

Comment:
can we use itertools combination function
i think we can avoid nested loop using this 
```suggestion
        # Compare each view model with every other view model to ensure their meta titles are different
        for (vm1, name1), (vm2, name2) in combinations(view_models, 2):
    self.assertNotEqual(
        vm1.website_meta_title,
        vm2.website_meta_title,
        f"{name1} and {name2} should have different meta titles"
    )
```
Something like this